airflow/www/views.py

```diff
@@ -282,9 +282,46 @@ def get_int_arg(value, default=0):
             paging=wwwutils.generate_pages(current_page, num_of_pages,
                                            search=arg_search_query,
                                            showPaused=not hide_paused),
-            auto_complete_data=auto_complete_data,
             num_runs=num_runs)
 
+    @expose('/dags_autocomplete')
+    @has_access
+    @provide_session
+    def dags_autocomplete(self, session):
+        arg_search_query = request.args.get('search', None)
+
+        if not arg_search_query:
+            return wwwutils.json_response({
+                'items': []
+            })
+
+        max_items_count = int(request.args.get('limit', 8))
```

Comment:
This is not needed at the moment, but I wanted to make the server code independent of the client's configuration. typeahead by default displays only 8 elements.